packages/webdriverio/tests/commands/browser/setWindowOuterSize.test.js

```diff
@@ -0,0 +1,67 @@
+import request from 'request'
+import { remote } from '../../../src'
+
+describe('setWindowOuterSize', () => {
+    let browser
+
+    beforeAll(async () => {
+        browser = await remote({
+            baseUrl: 'http://foobar.com',
+            capabilities: {
+                browserName: 'foobar'
+            }
+        })
+    })
+
+    it('should resize W3C browser window', () => {
+        browser.setWindowOuterSize(777, 888)
+        expect(request.mock.calls[1][0].method).toBe('POST')
+        expect(request.mock.calls[1][0].uri.path).toBe('/wd/hub/session/foobar-123/window/rect')
+        expect(request.mock.calls[1][0].body).toEqual({ x: null, y: null, width: 777, height: 888 })
+        request.mockClear()
```

Comment:
Can it be moved to afterEach ?

Reply:
If we move it to afterEach it will be called 7 more times without a reason, is it ok?

Reply:
Yes